corehq/apps/style/templates/style/crispy/multi_inline_field.html

```diff
@@ -0,0 +1,25 @@
+{% load crispy_forms_field %}
+{% load i18n %}
+
+{% if field.is_hidden %}
+    {{ field }}
+    {% if field.errors %}
+        <div class="alert alert-danger">
+            <strong>{% trans 'Errors' %}</strong>
+            {{ field.errors }}
+        </div>
+    {% endif %}
+{% else %}
+    {% if field|is_checkbox %}
+        <div id="div_{{ field.auto_id }}" class="checkbox">
+            <label for="{{ field.id_for_label }}" class="{% if field.field.required %} requiredField{% endif %}">
```

Comment:
@orangejenny yep, that's right. this template is basically just a merge/copy of https://github.com/dimagi/django-crispy-forms/blob/e0b51da82ba140426958491448014a1e9654a188/crispy_forms/templates/bootstrap3/layout/inline_field.html and https://github.com/dimagi/commcare-hq/blob/5e8f06e31e823e669260344786aea3240d54b325/corehq/apps/style/templates/bootstrap/layout/inline_field.html